spec/site/tlc/modes_spec.rb

```diff
@@ -107,7 +107,7 @@
     specify 'yellow flash can be activated with M0001', sxl: '>=1.0.7' do |example|
       Validator::Site.connected do |task,supervisor,site|
         prepare task, site
-        switch_yellow_flash
+        switch_yellow_flash 0
```

Comment:
i would prefer not to ad all these '0' arguments. let's use a defualt argument to avoid them

Reply:
i fixed this